cli/wallet/multisig.go

```diff
@@ -37,6 +42,16 @@ func newMultisigCommands() []cli.Command {
 				inFlag,
 			},
 		},
+		{
+			Name:      "send",
```

Comment:
I'm really unsure about `wallet send` command, it doesn't use wallet in any way, so it either should be something separate, or maybe `wallet sign` should send a transaction if it has enough signatures.

Reply:
My point was to make it explicit, to control whether tx is sent or not.
Maybe `--rpc` flag to sign will be good, just looks "dirty".